draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -485,6 +499,18 @@ queues via Differentiated Services Code Points (DSCP) or other mechanisms,
 which can substantially reduce the interplay with other traffic. However, a proposal
 targeting general Internet use can not assume this is always the case.
 
+As noted in {{circuit-breakers}}, all congestion control algorithms
+need to operate within the envelope set by network
+transport circuit breaker algorithms.
+{{!RFC8083}} also defines a
+minimal set of RTP circuit breakers. This identifies conditions under which a sender
+needs to stop transmitting media data to protect the network from excessive congestion.
+It is expected that, in the absence of long-lived excessive congestion
```

Comment:
Is there a missing comma at the end of this line? I would have expected something like:

  It is expected that, in the absence of long-lived excessive congestion,